tests/queries/0_stateless/02931_file_cluster.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+USER_FILES_PATH=$(clickhouse-client --query "select _path,_file from file('nonexist.txt', 'CSV', 'val1 char')" 2>&1 | grep Exception | awk '{gsub("/nonexist.txt","",$9); print $9}')
+
+for i in {1..10}
+do
+   echo \"file"$i"\","$i" > "$USER_FILES_PATH"/file"$i".csv
```

Comment:
Better to use file names unique for the test, because it can run in parallel with other tests. You can use `$CLICKHOUSE_TEST_UNIQUE_NAME` for it

Reply:
Yup, already see this -- the test is flaky, probably thanks to this :)